### PR TITLE
Add option for Elite Tracker (FR)

### PR DESCRIPTION
**Pull request by aureliensauv**
Add option for Elite Tracker (FR) for download torrent using https for tracker URL.

### CODE REVIEW

File: src/Jackett.Common/Indexers/EliteTracker.cs [hunk 1]
```diff
@@ -22,6 +23,7 @@ private string LoginUrl
         { get { return SiteLink + "takelogin.php"; } }
         private string BrowseUrl
         { get { return SiteLink + "browse.php"; } }
+        private bool TorrentHTTPSMode => ((ConfigurationDataEliteTracker)configData).TorrentHTTPSMode.Value;
 
         private new ConfigurationDataBasicLogin configData
```
Comment (kaso17) [inline src/Jackett.Common/Indexers/EliteTracker.cs]: Change this and line 30 to ConfigurationDataEliteTracker, then you shouldn't need the case in line 26.

File: src/Jackett.Common/Indexers/EliteTracker.cs [hunk 2]
```diff
@@ -206,6 +208,13 @@ public EliteTracker(IIndexerConfigurationService configService, WebClient webCli
                     release.Peers = ParseUtil.CoerceInt(Leechers.TextContent) + release.Seeders;
                     release.Grabs = ParseUtil.CoerceLong(Grabs.TextContent);
 
+                    if (TorrentHTTPSMode)
+                    {
+                        var linkHttps = Row.QuerySelector("td:nth-child(4)").QuerySelector("a").GetAttribute("href");
+                        var idTorrent = ParseUtil.GetArgumentFromQueryString(linkHttps, "id");
+                        release.Link = new Uri($"https://elite-tracker.net/download.php?id={idTorrent}&type=ssl");
```
Comment (kaso17) [inline src/Jackett.Common/Indexers/EliteTracker.cs]: Please use the SiteLink variable instead of hard coding https://elite-tracker.net/

**Comment by aureliensauv** [comment]
It's done!

**Comment by kaso17** [comment]
merged, thank you

**Pull request merged**